e2e/utils.go

```diff
@@ -745,3 +745,24 @@ func listSnapshots(f *framework.Framework, pool, imageName string) ([]snapInfo,
 	err := json.Unmarshal([]byte(stdout), &snapInfos)
 	return snapInfos, err
 }
+
+func GivePermToCephfsRoot(f *framework.Framework) {
+	opt := metav1.ListOptions{
+		LabelSelector: "app=rook-ceph-tools",
+	}
+	err := execCommandInPod(f, "mkdir /mnt/cephfs", rookNS, &opt)
```

Comment:
@poornimag can you make `/mntcephfs` as const 
Thanks for the Fix :)

Reply:
@poornimag one question do we need to create `/mnt/cephfs` ? or this will be created by rook operator ? and when this will be created

Reply:
> @poornimag can you make `/mnt/cephfs` as const
> Thanks for the Fix :)

Np:) Sure will do in the next update

Reply:
> @poornimag one question do we need to create `/mnt/cephfs` ? or this will be created by rook operator ? and when this will be created

This is created in tool pod for temperory mount of the cephfs filesystem. This happens before each in cephfs. Rook will not do this, this patch will be reverted once the actual fix is in place, so we need not do it in rook IMO.